app.py

```diff
@@ -0,0 +1,52 @@
+     
+class Order( ):
+    def __init__(self, orders = []):
+        self.orders = orders
+       
+        
+    def add_order(self, foodid, userid, date, status):
+
+        order1 = {
+            "id" : len(self.orders) + 1,
+            "foodid" : foodid,
+            "userid" : userid,
+            "date" : date,
+            "status" : status
+        }
+        self.orders.append(order1)
+        return order1
+
+    def get_order(self, id):
+        return self.search_order(id)
+
+        
+        
+    def remove_order(self, id):
+        for order in self.orders:
+            if order['id'] == id:
+                return self.orders.remove(order)
+            return ({'message':'list empty'})
+         
+        
+    def return_all_orders(self):
+        return self.orders
+         
```

Comment:
Too many white spacing in the code, you may try linting your work to remove unnecessary white spaces

Reply:
How do you run linting in vscode